test/specs/core.interaction.tests.js

```diff
@@ -79,83 +79,165 @@ describe('Core.Interaction', function() {
 	});
 
 	describe('index mode', function() {
-		it ('should return all items at the same index', function() {
-			var chart = window.acquireChart({
-				type: 'line',
-				data: {
-					datasets: [{
-						label: 'Dataset 1',
-						data: [10, 20, 30],
-						pointHoverBorderColor: 'rgb(255, 0, 0)',
-						pointHoverBackgroundColor: 'rgb(0, 255, 0)'
-					}, {
-						label: 'Dataset 2',
-						data: [40, 40, 40],
-						pointHoverBorderColor: 'rgb(0, 0, 255)',
-						pointHoverBackgroundColor: 'rgb(0, 255, 255)'
-					}],
-					labels: ['Point 1', 'Point 2', 'Point 3']
-				}
+		describe('intersect: true', function() {
+			it ('gets correct items', function() {
+				var chart = window.acquireChart({
+					type: 'line',
+					data: {
+						datasets: [{
+							label: 'Dataset 1',
+							data: [10, 20, 30],
+							pointHoverBorderColor: 'rgb(255, 0, 0)',
+							pointHoverBackgroundColor: 'rgb(0, 255, 0)'
+						}, {
+							label: 'Dataset 2',
+							data: [40, 40, 40],
+							pointHoverBorderColor: 'rgb(0, 0, 255)',
+							pointHoverBackgroundColor: 'rgb(0, 255, 255)'
+						}],
+						labels: ['Point 1', 'Point 2', 'Point 3']
+					}
+				});
+
+				// Trigger an event over top of the
+				var meta0 = chart.getDatasetMeta(0);
+				var meta1 = chart.getDatasetMeta(1);
+				var point = meta0.data[1];
+
+				var node = chart.canvas;
+				var rect = node.getBoundingClientRect();
+
+				var evt = {
```

Comment:
Might be easier and more accurate to generate a [Chart.js event](/chartjs/Chart.js/blob/master/src/platforms/platform.dom.js#L133) instead (`nativeEvent: null`)?